tests/mocks/mock_openai.py

```diff
@@ -0,0 +1,53 @@
+class mockCreateExceptionCommand:
+    pass
+
+
+class mockCreateException(Exception):
+    pass
+
+
+def get_mock_openai_class(original_class, create_responses, acreate_responses):
+    """
+    Unlike the mock mona clients, the class returned from this function
+    makes no assertions. We are not here to test calls to OpenAI, but
```

Comment:
That's not accurate... you have assertions in the create/acreate functions

Reply:
Fixed